osquery/tables/networking/windows/arp_cache.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#include <boost/range/algorithm/find.hpp>
+#include <osquery/tables.h>
+#include <string>
+
+#include "boost/algorithm/string/replace.hpp"
+#include "osquery/core/conversions.h"
+#include "osquery/core/windows/wmi.h"
+#include "osquery/tables/networking/windows/interfaces.h"
+
+namespace osquery {
+namespace tables {
+
+const std::map<unsigned short, const std::string> mapOfAddressFamily = {
```

Comment:
Typically with global variables we use the `kMapOfAddressFamily` naming convention